src/main/java/edu/harvard/iq/dataverse/DatasetServiceBean.java

```diff
@@ -127,7 +127,8 @@ public Dataset findDeep(Object pk) {
             .setHint("eclipselink.left-join-fetch", "o.files.dataFileTags")
             .setHint("eclipselink.left-join-fetch", "o.files.fileMetadatas")
             .setHint("eclipselink.left-join-fetch", "o.files.fileMetadatas.fileCategories")
-            //.setHint("eclipselink.left-join-fetch", "o.files.guestbookResponses")
+            .setHint("eclipselink.left-join-fetch", "o.files.fileMetadatas.varGroups")
```

Comment:
Is this part of the optimization here? I would guess this isn't often accessed, so I'm not sure how much of a performance help it would be.

Reply:
I had seen in in the query log even when it was not used. For large datasets it does make a difference (one query less for each file).